lib/Perl/Critic/Command.pm

```diff
@@ -518,7 +518,7 @@ sub _get_option_specification {
 
 sub _colorize_by_severity {
     my @violations = @_;
-    return @violations if _this_is_windows();
+    return @violations if _this_is_windows() && !eval 'require Win32::Console::ANSI; 1';
```

Comment:
I usually use stringy-eval for this because so many modules are now using auto-dependency scanners. The stringy version hides the dependency so that you don't have to explicitly deal with it somewhere else in the build process.